### PR TITLE
Small UI fixes and improvements

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Fixes issue where resizing shots moved them two timelines to the left.
- Fixes issue where some buttons in the Edit Project screen were unreadably small.
- Fixes issue where Listviews were incorrectly styled as if they were entirely selected.
- Lessens performance impact of Feedbackpane snapshots and blurring.
- Halves number of counts in timeline from 400 to 200, as temporary workaround. Setting the number of counts to 400 has a substantial performance impact on dragging for unknown reasons, and yes that's on a fast computer. EDIT: this is now an issue.
- Reintroduces dragging-blur, as this has virtually no performance impact.
- Styles DeleteCameraTypeWarningModalView
- Fixes issue where DeleteCameraTypeModalView could not display a too-large number of cameras.
- Removes some unused code from TimetableBlock class.

### CODE REVIEW

File: src/main/java/gui/modal/DeleteCameraTypeWarningModalView.java [hunk 1]
```diff
@@ -3,29 +3,64 @@
...
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.VBox;
 import lombok.Getter;
 
 public class DeleteCameraTypeWarningModalView extends ModalView {
     
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/modal/DeleteCameraTypeWarningModalView.java]: Whitespace

Comment (Markvanderuit) [inline src/main/java/gui/modal/DeleteCameraTypeWarningModalView.java]: Removed.

**Comment by bart-de-jonge** [comment]
Looks fine to me

**Comment by mennoo1996** [comment]
Good work !

**Pull request merged**